client/src/components/Main.tsx

```diff
@@ -131,12 +131,13 @@ const Part: React.FC = () => {
       headlessApi.current = api
       try {
         const p = storeApi.getState().examples.objs[storeApi.getState().activeExample].params
-        productOrSolidId.current = await create(api, p, { onSelect, onResume })
+        const id = await create(api, p, { onSelect, onResume })
+        productOrSolidIds.current = cad instanceof solid ? id as number[] : id as number
```

Comment:
Why is this change needed? The return type of await create(...) is the same as productOrSolidIds.current anyway

Reply:
You're right. I don't remember why I changed this code. I think I had somehow problems with the types. But it seems fine without the cast as well.
See [55f2cea](https://github.com/awv-informatik/buerli-examples/pull/11/commits/55f2cea068c459a65bacba60cea0c4af3010c575)